dashboard/test/controllers/api/v1/pd/workshops_controller_test.rb

```diff
@@ -89,6 +89,17 @@ class Api::V1::Pd::WorkshopsControllerTest < ::ActionController::TestCase
     assert_equal workshop_2.id, response[0]['id']
   end
 
+  test 'with the organizer view param, program managers can view workshops in their regional partner and workshops they organized' do
+    workshop_in_my_region = create :pd_workshop, regional_partner: @regional_partner, organizer: @workshop_organizer
+    sign_in @organizer
+
+    get :index, params: {organizer_view: 1}
+    assert_response :success
+    response = JSON.parse(@response.body)
+    assert_equal 2, response.length
+    assert_equal [@workshop.id, workshop_in_my_region.id], [response[0]['id'], response[1]['id']]
```

Comment:
Is the order preserved? It might be better to sort these to protect against future random orderings.

Reply:
Order is preserved. The `.where...or` orders by id, and the second workshop will always have a higher ID than the first

Reply:
Nevermind. Will order the query by id

Reply:
I don't know if we care that the query is ordered (though that's certainly a fine way to handle it). We could also sort the 2 arrays here in the comparison.